[Fixed #1372]IntroSplashScreen

iamsharad98 opened the pull request:
Add a introduction splash screen on starting of app.
### ScreenShot of changes done in-app:-
@therajanmaurya **Can you please review it ?**

![WhatsApp Image 2020-02-25 at 11 49 58 PM](https://user-images.githubusercontent.com/26202526/75276110-8d8eb780-582b-11ea-965f-71ee9aece345.jpeg)

dilpreet2028 committed:
feat:notification integration

Commit:
fix: Issue #473 - Given preference to Account Balance shown in Savings and Loan Account Details

Commit:
test: Unit Tests for AccountOverviewPresenter

se-ke committed:
test: Created tests for RegistrationVerificationPresenter

Commit:
test: Unit Test for ClientChargePresenter

therajanmaurya committed:
chore: update README

Commit:
fix: Issue #454 - Made  Button Background Borderless in Normal Transfer

Commit:
fix: Issue #453 - Cancel button working in Third Party Transfer

mofetej committed:
fix: remove none option from filter dialog

Remove Filter Dialog "None" Option

issue #466

Commit:
fix: Issue #451 - Showing Progress dialog while making Transfer

0xsiddharthks committed:
style: correct spelling mistake

manki11 committed:
fix:Opens Gmail on clicking contact email on home screen

ivary43 committed:
fix:Issue #484 -App is opening login Activity when pressed back in register activity

geekanamika committed:
fix : Minimum required balance label hidden on null data

AkshGautam committed:
Upload:Contributing.md

miPlodder committed:
Fixed inconsistency in Notification Number Box

miPlodder committed:
Fix: Added corect Marker Location and animation in the Help Section

ivary43 committed:
Fix: Fixed typos in Issue Template.

miPlodder committed:
fix: registration activity softkeyboard issue

AkshGautam committed:
Fix:Retaining state on Orientation changes

AkshGautam committed:
Fix:Zero value accepted as valid transfer

AkshGautam committed:
Fix:Empty spinner in absence of internet

AkshGautam committed:
Fix Issue#593:PR template

miPlodder committed:
fix: consecutive snackbars issue

Rish02Sharma committed:
fix:opens phone dialer on clicking helpline number

Commit:
fix: Issue #446 - End Date datepicker should be initialised to the date of start date

ivary43 committed:
fix: Fixed UI error in User Profile

miPlodder committed:
Style #606: added missing item closing tag </item>

miPlodder committed:
refractor: fixed typos and id's

mohak1712 committed:
fix: Next click on text input in AddBeneficiary screen

ivary43 committed:
Issue#558: Added error message in beneficiary section

miPlodder committed:
Fix #621 : Next button is changed to done button in forms

mohak1712 committed:
fix : DatePicker UI changes in Open Saving Account Transaction screen

mohak1712 committed:
fix : Notification icon on homescreen changes to circle

miPlodder committed:
Fix #625 : Transfer and 3rd party transfer confirmation fragment is scrollable till the bottom

ivary43 committed:
fix:Issue#628:filtered only spaces entry in mandatory fields

mohak1712 committed:
fix : Radio button changes to unchecked state when filtering transactions using dates

ivary43 committed:
fixIssue#629:Null is not shown as currency type in Recent transactions

sridharjajoo committed:
feat: Added animation to tick sign

Commit:
FixIssue#553:fixed  loan filtering for different account section

AkshGautam committed:
Fix: Improper Eye Icon behaviour in passcode activity

mohak1712 committed:
fix : Toolbar changes so that filter icon does not get cut

AkshGautam committed:
Fix:Improper error message in help,settings and about us

Commit:
Added validation for password length while registering client

AkshGautam committed:
Fix #660: Adding NA in empty Due Date and space between contact us and number

AkshGautam committed:
Feat #656:Adding scrollbar

miPlodder committed:
Fix #609 : Multiple Languages added in Settings

mohak1712 committed:
fix: Removes duplication from accounts in transfer screen

miPlodder committed:
Fix #663: User Image not showing

mohak1712 committed:
fix: adds remark during transfer

Rish02Sharma committed:
fix:check if the internet connection is active or not in RegistrationFragment

harshitbansal05 committed:
add fits system windows to navigation drawer

Commit:
fix:Setting emptyView if there is no data

miPlodder committed:
Fix #650: Softkeyboard blocks the UI

AkshGautam committed:
Fix #686:Action Bar Slides in case Add Beneficiary fragment

miPlodder committed:
fix #702: Inconsistent checks for Username

miPlodder committed:
Fix #685: App crashes on entering large values

miPlodder committed:
Fix #704: Allow spaces in between the password but don't allow in
start and end of the password

Rish02Sharma committed:
Migrate RxJava 1 to RxJava 2

therajanmaurya committed:
Fix: The ConfigurableReport.setDestination(Object) method has been deprecated and is scheduled to be removed in Gradle 5.0. Please use the method ConfigurableReport.setDestination(File) instead.

Commit:
refactor:Removed redundant type casting

Rish02Sharma committed:
Migrate RxJava1 to RxJava2 in test

dilpreet2028 committed:
feat: Option to change Endpoint configurations

Commit:
fix:added error message in case server is down

sridharjajoo committed:
fix: Removed the onBackPressed Bug in TransferProcessFragment

mohak1712 committed:
fix: added scroll to ensure visibility of withdraw loan button for longer text

miPlodder committed:
Fix #678 : Change visibility of ProgressBar on Fragment change

AkshGautam committed:
fix #731:Toolbar text getting changed and minimized on orientation change

ivary43 committed:
fix:added slide animation for opening of account overview

dilpreet2028 committed:
fix: saving instance of profile image

ankurs287 committed:
fix: App crashes in User Profile Section #729 [Updated ]

sridharjajoo committed:
fix: Keyboard persistence bug fixed

dilpreet2028 committed:
fix: classifying accounts according to their types

AkshGautam committed:
fix #740:formatting of text when language is switched to hindi

AkshGautam committed:
fix  #750: placing cursor at the end to save user click

miPlodder committed:
Fix #747 : Added Checks for empty Edittexts and for validity of Url in Update Endpoints Setting

miPlodder committed:
Fix #736 : User Profile UI Issue using Sweet Error Library[Offline Mode]

miPlodder committed:
Fix #764 : Added Numeric Softkeyboard for Account Number

ankurs287 committed:
fix: App crashes in User Profile Section #729 [Updated-2]

dilpreet2028 committed:
fix:remove loan accounts from PayFrom list

miPlodder committed:
Fix #775 : Softkeyboard blocks UI in Accounts

AkshGautam committed:
Fix #763: Adding butterknife support

AkshGautam committed:
fix #778:transfer within same account

mohak1712 committed:
fix : closed filter functionality

AkshGautam committed:
fix #751:Improving the UI of review fragment

ivary43 committed:
refactor:added dependency version in rootProject

sridharjajoo committed:
feat:Added proper formatting to the total deposits

miPlodder committed:
Fix #788 : Removed Latency Issues in Update Endpoints

sridharjajoo committed:
fix:Improved UI of home activity

miPlodder committed:
Fixed #791 : Syncronized Animation in Account Overview with other views

AkshGautam committed:
fix #739:Wrong toolbar text and navbar item

miPlodder committed:
Fix #793 : Resolved Nav Drawer selected item on Logout

miPlodder committed:
Fix #794 : User Profile directly shows Username

AkshGautam committed:
fix #799: Card view getting cut on orientation changes

miPlodder committed:
Style #804 : Correct TPT dialog message

miPlodder committed:
Fix #806 : Replaced Dialog Spinner in Add Beneficiary

AkshGautam committed:
fix #810:Replacing dialog spinner

miPlodder committed:
Fix #703 : Forms not loading when no Internet Connection [used Sweet Error Library]

miPlodder committed:
Style #807 : Corrected Msg on Transfer Confirmation Screen

miPlodder committed:
Fix 461: Enhancement in Accounts Section

miPlodder committed:
Fix #818 : Resizing Activities when Softkeyboard is active

ivary43 committed:
refactor:Added sweet error library across the app

miPlodder committed:
Fix #813 : Resolved Successful Transfer Crashes

miPlodder committed:
Fix #814 : Added proper descriptive message during Transfer process in all use-cases

miPlodder committed:
Fix #741 : Language Change Issue in Settings

miPlodder committed:
Fix #728 : App not crashing on Updating Loan

miPlodder committed:
Fix #829 : Integer Type Transfer Limit in Beneficiary send to Server

miPlodder committed:
Chore #525 : Added NDK ABI Filters

miPlodder committed:
Fix #659 : App crashes in TPT

miPlodder committed:
Fix #580 : Transfer are editable

ivary43 committed:
feat:added Qr import functionality to add beneficiary

miPlodder committed:
Feat #697 : Splash Screen Added

dilpreet2028 committed:
refractor: Add mifos passcode library

therajanmaurya committed:
Fix: open activity while adding beneficiaries

therajanmaurya committed:
Fix: performace while showing new local fab button

miPlodder committed:
Fix #579 : Visibility Icons changed in Account Overview

miPlodder committed:
Fix #859 : Added descriptive Beneficiary Spinner TPT

miPlodder committed:
Refactor #862: Converted Model classes to Kotlin Data Classes

miPlodder committed:
Chore #862: Updated Mockito Dependency to allow Mockito mock final Data Classes of Kotlin

miPlodder committed:
Fix #864 : Fixed Crash during Adding Beneficiary from Gallery

prashantkh19 committed:
Migration from GCM to FCM

miPlodder committed:
Feat #833: Added Support for Savings Account

miPlodder committed:
Fix #868: UI changed when Pending Savings Account is clicked

miPlodder committed:
Feat #839 : Support for Editing User Details

miPlodder committed:
Feat #839 : Support for Editing User Details

miPlodder committed:
Feat #835: Added Support for Guarantors

miPlodder committed:
Fix #874: Clicking Closed Share Account crash fixed

kalpesh30 committed:
Checking the list if it is null or not

m-murad committed:
fix: inconsistant navigation item selection in HomeActivity

etlhsu committed:
Fix #910: Unresolved dependencies error while building app

etlhsu committed:
Fix: #894: Progress Bar shown on top of Input Fields

migueldinisdev committed:
Fix #899: Replaced Snackbar with Toast in Recent Transactions

m-sameer committed:
Fix #895: Add error text below password EditText in LoginActivity

etlhsu committed:
docs: Updated README to reflect project

etlhsu committed:
Fix #897: Username is shown as null on login (#915)

migueldinisdev committed:
Fix #898: Submission/Disbursement date pickers

migueldinisdev committed:
Fix #931: Submitting Beneficiary Application form App crash

- Changed Transfer Limit datatype to Float in order to match the default value (and possible ones)
- Refactored Beneficary to Beneficiary package

migueldinisdev committed:
Fix #930: Removed ScrollView

letelete committed:
Feat: #943 Add polish translation

letelete committed:
Feat: Add spanish language translation

letelete committed:
Feat: Add kannada language translation

letelete committed:
Feat: Add indonesian language translation

letelete committed:
Feat: Add khmer language translation

m-sameer committed:
Fix #929: Fix NullPointerException in LoanAccountsTransactionPresenter (#941)

letelete committed:
Feat: Add french translation (#949)

migueldinisdev committed:
Fix #950: Added Portuguese translations (#951)

letelete committed:
Feat: Add bengali language translation (#954)

letelete committed:
Feat: Add swahili language translation (#953)

letelete committed:
Feat: Add russian language translation (#952)

m-sameer committed:
Feat: Add Urdu translations

etlhsu committed:
Fix #958: Transfer button size shrinks on Confirmation Page (#966)

m-sameer committed:
Feat: Add Burmese translations (#964)

m-sameer committed:
Feat: Add Arabic translations (#959)

letelete committed:
Fix: #935 Newly added beneficiary isn't visible until manually refresh (#937)

letelete committed:
Fix: #934 Back to the list activity after adding a new beneficiary (#936)

m-murad committed:
fix: unnecessary initialisation of HomeActivity again (#892)

Commit:
Fix #969: Language changes on Home Activity (#970)

luckyman20 committed:
fix: Issue #456 - Correct Values displayed at LoanAccountSummary (#971)

Commit:
Fix#456:Corrected previous PR for this issue (#974)

shiv07tiwari committed:
fix : Search Bar Allignment Issue

jsuyash1514 committed:
fix: Add check for null pointer exception on transaction list in saving account transaction fragment

rahul-jha98 committed:
Feat #988 Fab now appears and disappears with animation

rahul-jha98 committed:
fix #1000 Displaying currency code instead of null in Charges fragment

rahul-jha98 committed:
fix #1007: Prevented switching to home on clicking Share or Settings

shivangisingh1702 committed:
Fix:Add Beneficiary activity loses data on screen rotation

shivangisingh1702 committed:
fix #1020:Notification toolbar text changes on screen rotation

harrycode007 committed:
Added help label for help fragment

Commit:
fix #940: Removed XML errors and warnings

Commit:
Fixed Groovy warnings

RajaVamsi11 committed:
Update CONTRIBUTING.md

Fixed typos and some minor grammatical issues in contributing.md file

shiv07tiwari committed:
fix : share link

shiv07tiwari committed:
fix : dialog opens twice

rahul-jha98 committed:
fix #1009: New instance of home created only if language is changed (#1016)

Commit:
docs : Update README.md

Commit:
feat : Added Telugu translations

shiv07tiwari committed:
fix: logout highlighted after outside box click

shiv07tiwari committed:
fix: close dialog after user profile from navigation

shivangisingh1702 committed:
fix: #1048 Missing some bengali translation

Commit:
fix : Removed Hardcoded string in the menu_transfer.xml file

shiv07tiwari committed:
fix: reset dates on radio filter in Transaction Details

shiv07tiwari committed:
fix: add currency in loan summary and fix allignment

shiv07tiwari committed:
fix: currency before amount in savingaccountdetails (#1084)

shivangisingh1702 committed:
fix:Fixed grammer mistakes and spelling mistakes in portuguese (#1071)

jsuyash1514 committed:
feat: Enable search by account number (#982)

shiv07tiwari committed:
fix: currency before amount in loanRepayment

therajanmaurya committed:
Fix: Uncheck navigation drawer options for quick access features like Settings, About Us, Help etc

therajanmaurya committed:
Feat: Privacy Policy

shiv07tiwari committed:
fix: app crash in transaction review

shiv07tiwari committed:
fix: different messages on signup username

Commit:
fix: #1136 remove obsolete configurations

therajanmaurya committed:
Fix: findbugs check

therajanmaurya committed:
Fix: Parcelable models

therajanmaurya committed:
style: Update Build tools and support libraries version to latest for release

therajanmaurya committed:
Fix: Travis configuaration

therajanmaurya committed:
Fix: Intrumentation test setup according to new API

shiv07tiwari committed:
fix: double error message in qr scan

shiv07tiwari committed:
fix: extra data in qr share email

therajanmaurya committed:
refactor: Migration to AndroidX

therajanmaurya committed:
release: playstore release

therajanmaurya committed:
release: First Release on Playstore with updated package name org.mifos.mobile

aayushsingla committed:
Fix:Travis Build Error

prashantkh19 committed:
Fix: #1072 Updated Start and End date while selecting filter in account transactions

prashantkh19 committed:
Fix: #1056 Added functionality to open app with the device default language.

prashantkh19 committed:
Fix: #1068 Reformatted atring in the principal amount in loan application to show decimals upto 2 places.

Fixes travis build fail

prashantkh19 committed:
Fixes 886: Added support for lower api devices

Fixes travis build fail

shiv07tiwari committed:
fix: loan number in centre

prashantkh19 committed:
Fix #801 : Merge fix of PR 878

Fix #801 : Merge fix of PR 878

prashantkh19 committed:
Fix #808 : Merge fix of PR 812

prashantkh19 committed:
Fix #960 : Merge fix of PR 962

prashantkh19 committed:
Fix #1134 : Requested Changes in PR 1147

Fix #1117 : Merge fix in PR 1120

prashantkh19 committed:
Fix #1117 : Merge fix in PR 1120

prashantkh19 committed:
Fix #1104 : Merge fix in PR 1110

prashantkh19 committed:
Fixes #944 : Requested Changes in 961

prashantkh19 committed:
Fixes #1185: added support for loan application without loanPurposeId

checkpoint - fixed crash

prashantkh19 committed:
Feat: Added review screen for loan application

Updated strings.xml

prashantkh19 committed:
Fixes 1130: Replaced RecyclerView with TableView (#1182)

prashantkh19 committed:
updated filter in savingAccountsTransactionFragment (#1184)

updated filter in savingAccountsTransactionFragment

prashantkh19 committed:
Fixes #1185: added support for loan application without loanPurposeId (#1186)

prashantkh19 committed:
Added Unit Tests for HomeOldPresenter

prashantkh19 committed:
Added Unit Tests for GuarantorDetailPresenter, GuarantorListPresenter,
NotificationPresenter, SavingsAccountWithdrawPresenter,
UpdatePasswordPresenter and UserDetailsPresenter

prashantkh19 committed:
Integrated Payment-Hub to allow Third Party Transfers between different
fineract instances

geekanamika committed:
Revert "Integrated Payment-Hub to allow Third Party Transfers between different fineract instances"

This reverts commit efce97ea137b2b86a18cdce2d75c7f1520b6ad75.

elirehema committed:
Fix #1218 settings extra space (#1219)

devansh-299 committed:
fix#1126-button background follows app design (#1216)

devansh-299 committed:
Proper messages are shown when filtering returns nothing (#1200)

Commit:
Updated dependencies, libs and resolved errors (#1233)

laxyapahuja committed:
fix: date picker last date (#1245)

rchtgpt committed:
Fix: Add Material Dialog for login using password confirmation (#1238)

rchtgpt committed:
fix: remove hardcoded text from XML file (#1258)

dubeyaayush07 committed:
Removed error indicator when valid value is entered (#1350)

Fixes #1349

devansh-299 committed:
fix#1335 unusual behaviour of snackbar (#1336)

dubeyaayush07 committed:
No accounts found message is displayed when the search returns with zero matches (#1320)

Fixes #1319

devansh-299 committed:
fix#1312 - add change password fragment to backstack in settings fragment (#1313)

dubeyaayush07 committed:
Remove error layout when clear filter button is pressed (#1352)

Fix #1338

Commit:
Fixes #1353-Displaying null value in LoanAccountSummaryFragment

iamsharad98 committed:
Add a Intro Splash Screen Activity

anonghostteamz commented:
If you don't like the feature that I added on the splash screen then I will change it. My main concern is to add this feature. And I think it will definitely enhance the user experience.

devansh-299 commented:
@anonghostteamz app introduction has already been implemented in #1271 
If you want you can add/suggest your changes  there

Grandolf49 commented:
@anonghostteamz you have 3 checkstyle violation in your code. Check this file locally: **mifos-mobile/app/build/reports/checkstyle/checkstyle.html** to locate the exact details.
After solving them, run `gradlew check` in android terminal and make sure that build is succeeding

anonghostteamz commented:
Okay, @Grandolf49 I am working on it.

anonghostteamz commented:
@Grandolf49 Can you please tell me, Now I need to file PR for this issue again after editing or there is another option for this thing. Because Now my android studio gives me a warning that your access is denied and you are no longer do a pull request.

iamsharad98 committed:
Remove some extra string character.

iamsharad98 committed:
Add a introduction splash screen and remove extra string character that causes issue.

iamsh4shank commented on app/src/main/java/org/mifos/mobile/ui/activities/IntroSplashScreen.java:
```diff
@@ -0,0 +1,184 @@
+//        tvSkip = findViewById(R.id.tv_skip);
+
+        final List<ScreenItem> mList = new ArrayList<>();
+        mList.add(new ScreenItem("Bank account management", "With a bank account management feature, users can monitor their cards and bank account.", R.drawable.management));
```
Here the number of character are more than 100 in one line so add some character in the next line.

iamsh4shank replied:
Checkstyle doesn't accept more than 100 characters in one line so that this failed.  Just shift extra words in next line.

iamsh4shank commented on app/src/main/java/org/mifos/mobile/ui/activities/IntroSplashScreen.java:
```diff
@@ -0,0 +1,184 @@
+
+        final List<ScreenItem> mList = new ArrayList<>();
+        mList.add(new ScreenItem("Bank account management", "With a bank account management feature, users can monitor their cards and bank account.", R.drawable.management));
+        mList.add(new ScreenItem("Tracking spending habits", "Users can get more control over their finances and reach their saving goals." , R.drawable.track_money));
```
Here also

iamsh4shank commented on app/src/main/java/org/mifos/mobile/ui/activities/IntroSplashScreen.java:
```diff
@@ -0,0 +1,184 @@
+        final List<ScreenItem> mList = new ArrayList<>();
+        mList.add(new ScreenItem("Bank account management", "With a bank account management feature, users can monitor their cards and bank account.", R.drawable.management));
+        mList.add(new ScreenItem("Tracking spending habits", "Users can get more control over their finances and reach their saving goals." , R.drawable.track_money));
+        mList.add(new ScreenItem("Customer Support","Customer can get professional consultation or advice anytime they need it.", R.drawable.customer_support));
```
Here also more than 100 words issue and also here at 53 characters "," is not followed by white space.

iamsh4shank commented on app/src/main/java/org/mifos/mobile/ui/activities/IntroSplashScreen.java:
```diff
@@ -0,0 +1,184 @@
+
+                Intent mainActivity = new Intent(getApplicationContext(), LoginActivity.class);
+                startActivity(mainActivity);
+                // also we need to save a boolean value to storage so next time when the user run the app
```
Here also more than  100 characters in one line.

iamsh4shank commented on app/src/main/java/org/mifos/mobile/utils/IntroViewPagerAdapter.java:
```diff
@@ -0,0 +1,69 @@
+    @Override
+    public Object instantiateItem(@NonNull ViewGroup container, int position) {
+
+        LayoutInflater inflater = (LayoutInflater) mContext.getSystemService(Context.LAYOUT_INFLATER_SERVICE);
```
Here also 100 char issue

iamsh4shank commented on app/src/main/java/org/mifos/mobile/utils/IntroViewPagerAdapter.java:
```diff
@@ -0,0 +1,69 @@
+    public Object instantiateItem(@NonNull ViewGroup container, int position) {
+
+        LayoutInflater inflater = (LayoutInflater) mContext.getSystemService(Context.LAYOUT_INFLATER_SERVICE);
+        View layoutScreen = inflater.inflate(R.layout.layout_splash_intro,null);
```
Here at 75 character position  "," is not followed by whitespace.

iamsh4shank commented on app/src/main/java/org/mifos/mobile/utils/IntroViewPagerAdapter.java:
```diff
@@ -0,0 +1,69 @@
+    @Override
+    public void destroyItem(@NonNull ViewGroup container, int position, @NonNull Object object) {
+
+        container.removeView((View)object);
```
Here cast is not followed by white space.

iamsh4shank commented on app/src/main/java/org/mifos/mobile/utils/ScreenItem.java:
```diff
@@ -0,0 +1,37 @@
+
+public class ScreenItem {
+
+    String Title,Description;
```
Here at character position 18  "," is not followed by whitespace.

iamsharad98 committed:
Resolve CheckStyle Issue.

iamsharad98 committed:
Try Resolving CheckStyle Issue.

anonghostteamz commented:
Finally, I pass all the check pass. Thanks @robustTechie for your time and help.

robustTechie commented:
Your welcome @anonghostteamz :)

therajanmaurya commented on app/src/main/AndroidManifest.xml:
```diff
@@ -1,153 +1,150 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
-          xmlns:tools="http://schemas.android.com/tools"
-          package="org.mifos.mobile">
+    xmlns:tools="http://schemas.android.com/tools"
```
I see, there are so many changes that is not relevant to this PR. Please remove them.

therajanmaurya commented on app/build.gradle:
```diff
@@ -76,6 +76,8 @@ dependencies {
     implementation 'androidx.lifecycle:lifecycle-extensions:2.1.0'
     implementation 'androidx.lifecycle:lifecycle-viewmodel-ktx:2.1.0'
+    implementation 'androidx.appcompat:appcompat:1.1.0'
+    implementation 'androidx.constraintlayout:constraintlayout:1.1.3'
```
Add dependencies properly we below, you have to put the version number in versions.gradle and then use here. see below as an example.

iamsharad98 replied:
Okay, I will fix these issues and thank you for giving time to review my code.

iamsharad98 replied:
Can you please guide me on how to do this because I have not done it before.

therajanmaurya commented on app/src/main/java/org/mifos/mobile/ui/activities/IntroSplashScreen.java:
```diff
@@ -0,0 +1,191 @@
+        // make the activity on full screen
+//        requestWindowFeature(Window.FEATURE_NO_TITLE);
+//        getWindow().setFlags(WindowManager.LayoutParams.FLAG_FULLSCREEN,
+//                WindowManager.LayoutParams.FLAG_FULLSCREEN);
```
If these are not relevant then remove them.

therajanmaurya commented on app/src/main/java/org/mifos/mobile/ui/activities/IntroSplashScreen.java:
```diff
@@ -0,0 +1,191 @@
+
+        // hide the action bar
+
+        //getSupportActionBar().hide();
```
same as above.

therajanmaurya commented on app/src/main/java/org/mifos/mobile/ui/activities/IntroSplashScreen.java:
```diff
@@ -0,0 +1,191 @@
+
+    // show the GETSTARTED Button and hide the indicator and the next button
+    private void loaddLastScreen() {
+
```
remove this bank line.

therajanmaurya commented on app/src/main/java/org/mifos/mobile/ui/activities/IntroSplashScreen.java:
```diff
@@ -0,0 +1,191 @@
+    }
+
+    private void savePrefsData() {
+
```
remove this bank line.

iamsharad98 replied:
I want to ask you about this that, I have two option:-
1. The intro splash screen only shows once to the new user and when the second time user enters the app again then it will not show. (even when user not logged in on the first visit)
2. The intro splash screen shows every time to the new user until he logged in.
So, can you please tell me which idea is good?

therajanmaurya commented on app/src/main/java/org/mifos/mobile/ui/activities/IntroSplashScreen.java:
```diff
@@ -0,0 +1,191 @@
+
+    private boolean restorePrefData() {
+
+
```
remove these two bank line.

therajanmaurya commented on config/quality/checkstyle/checkstyle-config.xml:
```diff
@@ -1,7 +1,8 @@
 <!DOCTYPE module PUBLIC
     "-//Puppy Crawl//DTD Check Configuration 1.3//EN"
-    "http://www.puppycrawl.com/dtds/configuration_1_3.dtd">
+    "http://www.puppycrawl.com/dtds/configuration_1_3.dtd"
```
As these changes does not belongs to this PR so please revert back.

iamsharad98 replied:
I don't change anything over here. I just put this ">" in the next line by mistake. But I will definitely change it.

anonghostteamz replied:
As of now, I don't know kotlin but I will definitely look into it soon.

therajanmaurya commented:
@anonghostteamz As Pull Request template says to post screen shots if UI changes. Please add them and see my review comments and make the changes. I would suggest to convert java code into Kotlin, If you can't then It's fine.

devansh-299 commented:
> @anonghostteamz As Pull Request template says to post screen shots if UI changes. Please add them and see my review comments and make the changes. I would suggest to convert java code into Kotlin, If you can't then It's fine.

@therajanmaurya #1271 also solves the same problem, can you please review it?

anonghostteamz commented:
### ScreenShot of changes done in-app:-
@therajanmaurya **Can you please review it ?**

![WhatsApp Image 2020-02-25 at 11 49 58 PM](https://user-images.githubusercontent.com/26202526/75276110-8d8eb780-582b-11ea-965f-71ee9aece345.jpeg)

anonghostteamz commented:
@therajanmaurya As of now, I don't know kotlin so I keep this code in JAVA but I will definitely look into it soon.

iamsharad98 committed:
Edit IntroSplashScreen Activity

I add WindowSoftInputMode to "adjustResize" and change the position of IntroSplashScreen activity declaration.

iamsharad98 committed:
Revert back changes not related to this PR

Revert back all the changes that are not related to this PR and done by mistake(By Gradle).

iamsharad98 committed:
Revert Back Changes

This change is not related to this PR. So, I revert back it.

iamsharad98 committed:
Update IntroSplashScreen.java

Make code cleaner.

iamsharad98 committed:
Update build.gradle

Save versions in a different file.

iamsharad98 committed:
Update project/buid.gradle

jawidMuhammadi commented:
There are too many commits please rebase them in a single commit.

iamsharad98 committed:
Update UserProfileFragment.java

iamsharad98 committed:
Remove default photo for profile image.

iamsharad98 committed:
Update UserProfileFragment.java #2

iamsharad98 committed:
Merge remote-tracking branch 'origin/IntroSplashScreen' into IntroSplashScreen

# Conflicts:
#	app/build.gradle
#	app/src/main/java/org/mifos/mobile/ui/fragments/UserProfileFragment.java
#	app/src/main/res/values/colors.xml
#	app/src/main/res/values/strings.xml
#	build.gradle

ShivangiSingh17 commented:
@anonghostteamz Can you close this issue as the same issue has been addressed in #1271

rachittt commented:
Closing this PR as the commit history is disorganized, @anonghostteamz has deleted the branch and #1271 is working on the same issue.
